webapp/src/blocks.tsx

```diff
@@ -1122,6 +1136,23 @@ export class Editor extends toolboxeditor.ToolboxEditor {
                 blockXml: `<block type="pxt-on-start"></block>`
             });
         }
+        // Inject "Create a Sprite..." button
+        if (ns == "sprites") {
+            // TODO(dz):
+            extraBlocks.push({
+                name: `SPRITE_WIZARD_BUTTON`,
+                type: "button",
+                attributes: {
+                    blockId: `SPRITE_WIZARD_BUTTON`,
+                    label: "Create a Sprite...",
+                    weight: 101,
```

Comment:
Can we add a comment about the weight and why it's this value? Otherwise 101 just looks like a magic number :)

Reply:
I added a comment to clarify.